code/SortImports.hs

```diff
@@ -1,25 +1,28 @@
-{-# LANGUAGE BangPatterns, LambdaCase, OverloadedStrings, RecordWildCards #-}
+{-# LANGUAGE BangPatterns      #-}
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards   #-}
 {-# OPTIONS_GHC -Wall #-}
 module Main (main) where
 
-import Control.Applicative
-import Control.Exception
-import Control.Monad
-import Data.Char
-import Data.Functor
-import Data.List
-import Data.Maybe
-import Data.Monoid
-import Data.Set (Set)
-import Data.Text (Text)
-import System.Directory
-import System.Environment
-import System.Exit
-import System.IO (stderr, hPutStrLn)
+import           Control.Applicative
+import           Control.Exception
+import           Control.Monad
 import qualified Data.Attoparsec.Text as P
-import qualified Data.Set as S
-import qualified Data.Text as T
-import qualified Data.Text.IO as T
+import           Data.Char
+import           Data.Functor
+import           Data.List
+import           Data.Maybe
+import           Data.Monoid
+import           Data.Set             (Set)
+import qualified Data.Set             as S
+import           Data.Text            (Text)
+import qualified Data.Text            as T
+import qualified Data.Text.IO         as T
+import           System.Directory
+import           System.Environment
+import           System.Exit
+import           System.IO            (hPutStrLn, stderr)
```

Comment:
No particular reason, I just ran `stylish-haskell` on that script. Feel free to skip the first commit.